book/opmanual_autolab/50_autolab_operation_manual/51_autolab_HW_checks_script.md

```diff
@@ -0,0 +1,117 @@
+IMPORTANT: the SD card or USB memory stick must be formatted as `ext4`. You can change this by plugging in the memory stick or SD card in the computer, choose `format` and then choose `ext4`.
+
+### Step 3
+
+Check if the standoff on the Duckiebot is mounted and an April tag matching the Duckiebot name (hostname) is attached. The April tag needs to be aligned with the front of the top plate and centered. On the top plate you will find engraved lines that show you where to position it.  the standoff needs to be mounted in the very front of the bot. A picture can be found in [](#fig:standoffs-mounted-autobot)
+
+### Step 4
+
+Check if a rubber duckie sits on the bot. If not, put one on it now.
+As the rubber duckies are rather bad at balancing and lack hands to hold on to the Duckiebot, use double sided tape to fix it to the Duckiebot.
+
+### Step 5
+
+Make sure the back bumper is properly fastened to the Duckiebot (no wiggling allowed).
+
+### Step 6
+
+Check that no cables can interfere with the wheels of the Duckiebot. If necessary, use zip-ties to fix everything properly.
+
+### Step 7
+
+Check that all cables are properly attached. Especially check that the camera cable is plugged in correctly and locked.
+
+### Step 8
+
+If all above steps are done, press <kbd>y</kbd> and <kbd>ENTER</kbd> on your keyboard
+
+### Step 9
+
+In ETH, we currently use batteries that are no longer produced. To replace them, we manufacture special batteries just for Duckietown which you will have gotten with your Duckiebot. So if you use these that's even better. To be able to write down what battery you use, press <kbd>n</kbd> and <kbd>ENTER</kbd>. This will give you the possibility to type in what battery you use.
+
+### Step 10
+
+Check that a standard actuators (yellow motors) are used to drive the Duckiebot. If that is the case, press  <kbd>y</kbd> and <kbd>ENTER</kbd> on your keyboard, otherwise press <kbd>n</kbd> and <kbd>ENTER</kbd>. Afterwards describe the other motors the Duckiebot is using.
+
+### Step 11
+
+Fill in your name and press <kbd>ENTER</kbd>, the container should now close without errors.
+
+### Step 12
+The freshly created suitability test report can be found under ```http://HOSTNAME.local:8082/config/YYYY-MM-DD_hardware-compliance.yaml```
+
+### Step 13
+
+Now upload the file that was created to your fleet-roster. If you don't know what a fleet-roster is and how to create one, read [this](#autolab-fleet-roster). Once you created your own fleet-roster, upload your newly created file to it.
+
+Note: If you are using this instruction to operate the Autolab in ETHZ, upload the files to ```https://github.com/duckietown/ETHZ-autolab-fleet-roster/tree/aido2/autobots/autobotXX/hardware-compliance/YYYY-MM-DD_hardware-compliance```. where you need to replace the `XX` and the `YYYY-MM-DD`.
```

Comment:
remove this and just say to add it in a compliant way described in the fleet roster page please

Reply:
done